test/e2e/atlas/kubernetes_config_generate_test.go

```diff
@@ -95,19 +95,19 @@ func InitialSetupWithTeam(t *testing.T) KubernetesConfigGenerateProjectSuite {
 	s := KubernetesConfigGenerateProjectSuite{
 		t: t,
 	}
-	s.generator = newAtlasE2ETestGenerator(s.t)
+	s.generator = newAtlasE2ETestGenerator(t)
```

Comment:
given these are pointers, there should be no difference, have you noticed an improvement?

Reply:
the main change was https://github.com/mongodb/mongodb-atlas-cli/pull/1863/files#r1165638580 and then out of cautious decided to make sure to to use `s.t` too much but you are right, in theory being a pointer should be the same